Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/EntityPropertyEditor.cpp

```diff
@@ -2718,6 +2740,8 @@ namespace AzToolsFramework
             command->SetParent(m_currentUndoOperation);
 
             AfterPropertyModified(node);
+            SetPropertyEditingComplete(node);
```

Comment:
When does ContextMenuActionSetDataFlag() get called? Why are we only adding the EditingComplete() call here instead of in AfterPropertyModified()?

Reply:
I'm not sure when ContextMenuActionSetDataFlag() is called. It looks like slice code to me.

Typically the BeforePropertyModified() and AfterPropertyModified() events are sent from the DPE (part of IPropertyEditorNotify class). They are sent multiple times during editing (ex. moving a slider) while EditingComplete() is sent once at the end of editing. Here in ContextMenuActionSetDataFlag(), it's calling the handlers explicitly, so to have things working the same way, we expect a BeforePropertyModified -> AfterPropertyModified -> BeforePropertyModified -> AfterPropertyModifed -> ... -> EditingComplete.